src/components/explorer/utils.js

```diff
@@ -100,7 +100,7 @@ export const useDatasetResources = (datasetIDOrDatasetIDs) => {
             const r = Object.values(
                 Object.fromEntries(
                     datasetIDs
-                        .flatMap(d => datasetResources[d]?.data ?? [])
+                        .flatMap(d => Object.values(datasetResources[d]?.data ?? []))
```

Comment:
I don't really understand this - it worked before, and dataset resources should be a list of resources to my knowledge.

Reply:
it can also be an object, but then it doesn't match with the metadata resources field defined in phenopackets - if it truly is an object, the default value should be `{}` not `[]`

Reply:
The issue was not the hook after all, but the reducer. 
I removed this change and corrected the reducer so that it handles arrays properly.